### PR TITLE
doc: Documentation edits for 2.7.99-cs1 release

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added firmware bundle compatibility table.
Updated nRF54H20 GS guide.
Added initial power management documentation.
Added initial clock control documentation

### CODE REVIEW

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+The firmware can choose to keep each clock domains enabled even when all its sinks are inactive.
+
+Clock domain source selection differs between clock domains:
+
+* The LFCLK source is selected by the System Controller Firmware.
+* The HFXO source is selected by the System Controller Firmware.
+* The FLL16M source is selected by local domains or System Controller Firmware (SCFW) in all scenarios, except one:
+  When the firmware selects open-loop, but any of the HSFLL sources is in closed-loop, the FLL16M is automatically switched by the hardware to closed-loop.
+
+* HSFLLs sources are selected by firmware (depending on an HSFLL instance: local domain or System Controller).
```
Comment (divipillai) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:43]:
```suggestion
* The LFCLK source is selected by the System Controller Firmware (SCFW).
* The HFXO source is selected by the SCFW.
* The FLL16M source is selected by local domains or SCFW in all scenarios, except when the firmware selects open-loop, but any of the HSFLL sources are in closed-loop, the FLL16M is automatically switched by the hardware to closed-loop.
* HSFLL sources are selected by firmware (depending on an HSFLL instance, which is local domain or System Controller).
```

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The application-facing APIs expose only the domain that directly clocks the component used by the application.
+The management of the clocks on which this domain depends is handled internally by the clock driver associated with this domain.
+
+For example, a firmware module might request better timing accuracy for a fast UART instance.
+In this case, the firmware module requests the accuracy from the global HSFLL device driver, which directly clocks the UART.
+The dependencies, such as FLL16M and LFXO, are managed internally by the global HSFLL driver, not directly by the firmware module.
+
+
```
Comment (divipillai) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:52]:
```suggestion
```
extra line.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+LFCLK
+=====
+
+The Low-Frequency Clock domain is responsible for generating a 32768 Hz clock signal for ultra-low-power peripherals like ``GRTC`` or ``RTC``.
+
+HFXO
+====
+
+The High-Frequency Crystal Oscillator domain is responsible for clocking peripherals requiring high accuracy and MHz range frequency.
```
Comment (divipillai) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:61]:
```suggestion
The high-frequency crystal oscillator domain is responsible for clocking peripherals requiring high accuracy and MHz range frequency.
```

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst (hunk 1 above)
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:43]: I would rephrased this part to somethinkg like

"
Clock domain sources are selected by clock_control drivers based on the clock parameters' requirements reported by the software modules using the clock_control functions. The requirements may cover clock frequency, accuracy, or precision. Some clock domains are configured by clock_control drivers using LRCCONF peripherals, others are configured with assistance of the System Controller Firmware.
"

I'm not sure if more details is required in the public documentation.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Local HSFLLs
+============
+
+Local HSFLLs are clocking CPUs in local domains, fast peripherals around them, and local RAM.
+
+Global HSFLL
+============
+
+Global HSFLL clocks "fast" peripherals, global CPUs, RAM, and MRAM blocks.
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:84]:
```suggestion
Global HSFLL clocks "fast" peripherals, FLPR, RAM, and MRAM blocks.
```

I guess "global CPUs" is not a broadly understood term

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Zephyr clock control API
+************************
+
+Zephyr RTOS contains a ``clock_control`` device driver class for managing clocks.
+The ``clock_control`` API is designed to support multiple requestors.
+Moreover, the ``clock_control`` API supports blocking or asynchronous operations based on notifications when the requested clock settings are applied.
+Because of this, the ``clock_control`` API is well-suited for LFCLK management in the local domains.
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:92]:
```suggestion
```

This sentence is a justification of the design decision. It doesn't provide user-related information.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+The ``clock_control`` subsystem exposes portable parameters in its functions:
+
+* Accuracy requests in ppm values
+* Precision requests in ppm over n cycles, period-period jitter, or (for simplicity) high- and low-precision modes
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:97]:
```suggestion
* Precision requests in enumerated high- and low-precision modes
```

The non-architectural design of the driver selected this option

File: doc/nrf/samples/other.rst [hunk 7]
```diff
@@ -12,3 +12,4 @@ Other samples
    ../../../samples/ipc/*/README
    ../../../samples/mpsl/*/README
    ../../../samples/benchmarks/*/README
+   ../../../samples/benchmarks/multicore/*/README
```
Comment (divipillai) [inline doc/nrf/samples/other.rst:15]:
```suggestion
   ../../../test/benchmarks/multicore/*/README
```

I think the path is incorrect.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+
+Power states details
+********************
+
+The following sections describes the details of each of the software power states available on the nRF54H20 SoC.
+
+Active
+======
+
+*Active* is the power state in which the CPU is actively processing.
```
Comment (divipillai) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst:43]: Should the power states be made italic. I haven't see these mentioned as Italic in other docs.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+   * - CPU
+     - Powered on but not clocked.
+       Its state is retained in the hardware flip-flops.
+
+   * - RAM
+     - Enabled or retained depending on the hardware state.
+       RAM in caches is disabled.
+
+   * - System state
+     - * *System ON* if at least one other CPU or a peripheral clocked > 32 kHz is active.
```
Comment (divipillai) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst:166]:
```suggestion
     - * *System ON* if at least one other CPU or a peripheral clocked greater than 32 kHz is active.
```

**Comment by FrancescoSer** [comment]
@hubertmis @divipillai Implemented most of your suggestions, and added additional sections in the architecture_pm document. While I'll implement the remaining suggestions, please feel free to rereview that document.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+
+   * - System timer (based on GRTC)
+     - Active
+
+The power consumption in this state is lower than in *Active*, but higher than in other power states.
+The wake-up latency is higher than in *Active*, but lower than in the other power states.
+The primary sources of wake-up latency are the wake-up of the ``MRAMC`` and the startup of clock sources.
+The maximum latency depends on the MRAM power state configured by the System Controller.
+
+Idle with cache retained in RAM
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst:102]: Now that we don't list to-be-implemented states, I think we should remove them from this section

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+---------------------
+
+The HFXO, like any other clock domain, can be connected in hardware to one of the available sources.
+And it provides the clock signal to all of its sinks.
+
+FLL16M
+======
+
+The FLL16M clock domain clocks most of the peripherals in the system ("slow", 16 MHz).
+Its accuracy results in the timing accuracy of slow ``UART``, ``SPI``, ``TWI``, ``PWM``, ``SAADC``, etc.
```
Comment (annwoj) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:68]:
```suggestion
Its accuracy results in the timing accuracy of slow ``UART``, ``SPI``, ``TWI``, ``PWM``, ``SAADC``, among others.
```

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+Global HSFLL clocks "fast" peripherals, FLPR, RAM, and MRAM blocks.
+
+Zephyr clock control API
+************************
+
+Zephyr RTOS contains a ``clock_control`` device driver class for managing clocks.
+The ``clock_control`` API is designed to support multiple requestors.
+Moreover, the ``clock_control`` API supports blocking or asynchronous operations based on notifications when the requested clock settings are applied.
+
+The ``clock_control`` subsystem exposes portable parameters in its functions:
```
Comment (annwoj) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:87]:
```suggestion
The ``clock_control`` subsystem exposes portable parameters in its functions, which include:
```

**Review by annwoj — COMMENTED** [review]
just some general suggestions

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst (hunk 8 above)
Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst:36]: The problem I see is with power states having a long name like "Local suspend to Ram". Not highlighting it in some way might render sentences containing it quite difficult to read. Not necessarily they have to be in Italics, but I believe that they should be highlighted.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+The firmware can choose to keep each clock domains enabled even when all its sinks are inactive.
+
+Clock domain sources are selected by ``clock_control`` drivers based on the clock parameter requirements reported by the software modules using the ``clock_control`` functions.
+These requirements may include clock frequency, accuracy, or precision.
+Some clock domains are configured by ``clock_control`` drivers using ``LRCCONF`` peripherals, while others are configured with the assistance of the System Controller Firmware.
+
+The application-facing APIs expose only the domain that directly clocks the component used by the application.
+The management of the clocks on which this domain depends is handled internally by the clock driver associated with this domain.
+
+For example, a firmware module might request better timing accuracy for a fast UART instance.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:43]: @hubertmis Could you check this?

File: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst [hunk 14]
```diff
@@ -0,0 +1,52 @@
+.. _migration_nrf54h20_to_2.7.99-cs1:
```
Comment (FrancescoSer) [inline doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst:1]: @kenr Could you review this?

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 15]
```diff
@@ -81,10 +81,10 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
 
 .. _ug_nrf54h20_install_toolchain:
 
-Installing the |NCS| v2.7.0 and its toolchain
-*********************************************
+Installing the |NCS| v2.7.99-cs1 and its toolchain
+**************************************************
 
-You can install the |NCS| v2.7.0 and its toolchain by using Toolchain Manager.
+You can install the |NCS| v2.7.99-cs1 and its toolchain by using Toolchain Manager.
```
Comment (kenr) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:87]: @jangalda-nsc , the v2.7.99-cs1 is published?

File: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+  ``nrfutil device`` has been updated to version 2.4.6.
+
+  Install the nRF Util ``device`` command version 2.4.6 as follows::
+
+     nrfutil install device=2.4.6 --force
+
+  For more information, consult the `nRF Util`_ documentation.
+
+SDFW and SCFW firmware bundle
+  The SDFW and SCFW firmware bundle has been updated to version 0.6.2.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst:21]: It's one bundle, the are both in one package

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 17]
```diff
@@ -180,9 +180,9 @@ Using the nRF54H20 DK with the |NCS| v2.7.0 requires the following:
 
       nrfutil self-upgrade
 
-#. Install the nRF Util ``device`` command to version 2.4.0 as follows::
+#. Install the nRF Util ``device`` command version 2.4.6 as follows::
 
-      nrfutil install device=2.4.0 --force
+      nrfutil install device=2.4.6 --force
```
Comment (kenr) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: See above commane

Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: @kenr which comment? Could you be more explicit?

**Review by ns-tolu — APPROVED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst (hunk 17 above)
Comment (kenr) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: The entire document is using nrfutil-device 2.4.6, if we upgrade this one place we need to upgrade here also.
The latest version of nrfutil-device is currently 2.5.0.

Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: Ok, clarified with @shanthanordic , the version is 2.4.6

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+* The following calls in the `Zephyr's nRF clock control API extensions`_ (:file:`include/zephyr/drivers/clock_control/nrf_clock_control.h`):
+
+  * ``nrf_clock_control_request()``: Requests a reservation to use a given clock with specified attributes.
+  * ``nrf_clock_control_release()``: Releases a reserved use of a clock.
+  * ``nrf_clock_control_cancel_or_release()``: Safely cancels a reservation request.
+
+* The following calls in the `clocks devicetree macro API`_ (:file:`include/zephyr/devicetree/clocks.h`):
+
+  * ``DT_CLOCKS_CTLR_BY_IDX()``: Gets the node identifier for the controller phandle from a *clocks* phandle-array property at an index.
+  * ``DT_CLOCKS_CTLR()``: It is equivalent to ``DT_CLOCKS_CTLR_BY_IDX()``.
```
Comment (bjarki-andreasen) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:113]: `* ``DT_CLOCKS_CTLR()``: It is equivalent to ``DT_CLOCKS_CTLR_BY_IDX()`` with index (idx) set to 0.`

**Review by bjarki-andreasen — APPROVED** [review]
Nice work :) I only have two minor nits

**Review by divipillai — APPROVED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 19]
```diff
@@ -214,17 +214,17 @@ To prepare the nRF54H20 DK for first use, you must manually program the values o
 
 #. Move the BICR HEX file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
```
Comment (kl-cruz) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:215]: I do not think that this step is required. This command can be executed from ``Downloads`` directory.
Additionally - Can we program BICR after switching to ROT LCS?

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst (hunk 17 above)
Comment (nordic-piks) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: I just did those steps at windows and get:
```
Downloads\nrfutil>nrfutil.exe  --version
nrfutil 7.6.0 (92ce3a0 2023-08-15)
commit-hash: 92ce3a0c1cc7e203326d85919d826009510dc631
commit-date: 2023-08-15
host: x86_64-pc-windows-msvc
build-timestamp: 2023-08-23T08:25:25.3155777Z
classification: nrf-external


Downloads\nrfutil>nrfutil self-upgrade
Error: Could not update package registry

Caused by:
    0: failed to fetch `https://github.com/nordicsemiconductor/nrfutil-package-index`
    1: network failure seems to have happened
       if a proxy or similar is necessary, setting the environment variable `NRFUTIL_NET_GIT_FETCH_WITH_CLI` to `true` may help here

    2: failed to set hostkey preference: The requested method(s) are not currently supported; class=Ssh (23)
    
   Downloads\nrfutil>nrfutil install device=2.4.6 --force
Error: Could not update package registry

Caused by:
    0: failed to fetch `https://github.com/nordicsemiconductor/nrfutil-package-index`
    1: network failure seems to have happened
       if a proxy or similar is necessary, setting the environment variable `NRFUTIL_NET_GIT_FETCH_WITH_CLI` to `true` may help here

    2: failed to set hostkey preference: The requested method(s) are not currently supported; class=Ssh (23)

```

Can someone confirm?

Comment (nordic-piks) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:185]: Works at linux (WSL)

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst (hunk 19 above)
Comment (nordic-piks) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:215]: Yes, BICR can be programmed at LCS ROT.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 20]
```diff
@@ -214,17 +214,17 @@ To prepare the nRF54H20 DK for first use, you must manually program the values o
...
 
 Programming the SDFW and SCFW
 =============================
 
-After programming the BICR, the nRF54H20 SoC requires the provisioning of a bundle ( :file:`nrf54h20_soc_binaries_v0.5.0.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
+After programming the BICR, the nRF54H20 SoC requires the provisioning of a bundle ( :file:`nrf54h20_soc_binaries_v0.6.2.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
 To program the Secure Domain Firmware (SDFW, also known as ``urot``) and the System Controller Firmware (SCFW) from the firmware bundle to the nRF54H20 DK, do the following:
 
-1. Download the `nRF54H20 firmware bundle v0.5.0`_.
+1. Download the `nRF54H20 firmware bundle v0.6.2`_.
```
Comment (nordic-piks) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:227]: What about instruction for upgrade bundle 0.5.0 -> 0.6.2? Is that needed here?
Command is different.

Comment (nordic-piks) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:227]: Already find it.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst (hunk 13 above)
[the remark this answers is not in the document]
Comment (hubertmis) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_clockman.rst:44]: Not sure. In IPS I can find a term "Fast clock domain", but nothing directly about peripherals. The "fast UART" is the one in the "fast clock domain".

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst [hunk 21]
```diff
@@ -0,0 +1,247 @@
...
+The primary sources of wake-up latency are the wake-up of the ``MRAMC`` and the startup of clock sources.
+The maximum latency depends on the MRAM power state configured by the System Controller.
+
+Local Suspend to RAM
+********************
+
+*Local Suspend to RAM* is a sleep state that balances between power consumption and wake-up latency.
+
+This state is available only for DVFS-capable domains.
+Other domains, such as Radio, cannot retain the CPU and local peripherals in the *System ON (All) Idle* state.
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst:108]: This looks like typo.
```suggestion
Other domains, such as Radio, can retain the CPU and local peripherals in hardware flip-flops in the *System ON (All) Idle* state.
```

**Review by hubertmis — APPROVED** [review]

File: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+        On MacOS, connecting the DK might cause a popup containing the message ``Disk Not Ejected Properly`` to repeatedly appear on screen.
+        To disable this, run ``JLinkExe``, then run ``MSDDisable`` in the J-Link Commander interface.
+
+  #. List all the connected development kits to see their serial number (matching the one on the DK's sticker)::
+
+        nrfutil device list
+
+  #. Move the BICR HEX file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
+
+        nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware <path_to_bicr.hex> --core Secure --serial-number <serialnumber>
```
Comment (nordic-piks) [inline doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst:61]: --core Application
@FrancescoSer That needs to be updated too.
With Secure and device already provisioned (LCS ROT), it will fail.
